code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -84,14 +94,18 @@ AI MODULES
 /obj/item/aiModule/safeguard
 	name = "\improper 'Safeguard' AI module"
 	var/targetName = ""
-	desc = "A 'safeguard' AI module: 'Safeguard <name>. Anyone threatening or attempting to harm <name> is no longer to be considered a crew member, and is a threat which must be neutralized.'."
```

Comment:
Unless I somehow have dementia before my 20s this seems to be removing a big chunk of the safeguard module's law.

Reply:
Was removing description changes because now using it in hand shows laws. Might have accidentally removed the main description, gotta fix it.

Reply:
Wait, no, I didn't. All that stuff is now in the show_laws proc.

Reply:
If that is the case, please remove the description from all of the other lawsets, otherwise, the safeguard description is the odd one out.

Reply:
Description tells you the general idea of the lawset, not the laws they actually contain. There is no need to change description that says "A 'safeguard' AI module: 'Safeguard TARGET.".

Reply:
I mean in regards to changing the descriptions of the other modules. With this change only the safeguard module has had its laws removed from the description; you should change the other law modules as well to maintain consistency.

https://github.com/vlggms/tegustation-bay12/blob/76a4b1051584d12e97d62883d0b5f7b9d7f92636/code/game/objects/items/weapons/AI_modules.dm#L159

https://github.com/vlggms/tegustation-bay12/blob/76a4b1051584d12e97d62883d0b5f7b9d7f92636/code/game/objects/items/weapons/AI_modules.dm#L170

https://github.com/vlggms/tegustation-bay12/blob/76a4b1051584d12e97d62883d0b5f7b9d7f92636/code/game/objects/items/weapons/AI_modules.dm#L181

https://github.com/vlggms/tegustation-bay12/blob/76a4b1051584d12e97d62883d0b5f7b9d7f92636/code/game/objects/items/weapons/AI_modules.dm#L192

https://github.com/vlggms/tegustation-bay12/blob/76a4b1051584d12e97d62883d0b5f7b9d7f92636/code/game/objects/items/weapons/AI_modules.dm#L203

etc.etc.

Reply:
Oh right, will do.